src/site/markdown/server/manage-webadmin.md

```diff
@@ -1754,6 +1755,108 @@ Response codes:
  - 404: The mail queue does not exist
  - 500: Internal error
 
+## Administrating DLP Configuration
+Webadmin allows Data Lost Prevention (DLP) configuration
+
+- [List DLP configuration by sender domain](List_dlp_configuration_by_sender_domain)
+- [Store DLP configuration by sender domain](Store_dlp_configuration_by_sender_domain)
+- [Remove DLP configuration by sender domain](Remove_dlp_configuration_by_sender_domain)
+
+### List DLP configuration by sender domain
+
+Retrieve a DLP configuration for corresponding `senderDomain`, a configuration contains list of configuration item
+```
+curl -XGET http://ip:port/dlp/rules/senderDomain
+```
+
+Response codes:
+
+ - 200: A list of dlp configuration items is returned
+ - 400: Invalid senderDomain or payload in request
+ - 404: The domain does not exist.
+ - 500: Internal error
+
+```
+{"rules : [
+  {
+    "id": "1",
+    "expression": "james.org",
+    "explanation": "Find senders or recipients containing james[any char]org",
+    "targetsSender": true,
+    "targetsRecipients": true,
+    "targetsContent": false
+  },
+  {
+    "id": "2",
+    "expression": "Find senders containing apache[any char]org",
+    "explanation": "apache.org",
+    "targetsSender": true,
+    "targetsRecipients": false,
+    "targetsContent": false
+  }
+]}
+```
+
+### Store DLP configuration by sender domain
+
+Store DLP a configuration for corresponding `senderDomain`, if any item of DLP configuration in the request is stored before, 
+it will not be stored again
+
+```
+curl -XPUT http://ip:port/dlp/rules/senderDomain
+```
+
+The body can containing a list of DLP configuration items formed by those fields: 
+- `id`(String) is mandatory
+- `expression`(String) is mandatory
+- `explanation`(String) is optional
+- `targetsSender`(boolean) is optional default false
```

Comment:
s/is optional default/is optional and defaults to/

That would sounds more English to me.